db/migrate/20230316180217_set_call_logs_js_context_as_binary.rb

```diff
@@ -0,0 +1,11 @@
+class SetCallLogsJsContextAsBinary < ActiveRecord::Migration
+  def up
+    connection.execute "UPDATE call_logs SET js_context = NULL WHERE js_context IS NOT NULL"
+    change_column :call_logs, :js_context, :binary
+  end
+
+  def down
```

Comment:
The `up` and `down` funcs here have the very same code, `down` here shouldn't do the opposite?